lib/opengraph-io.rb

```diff
@@ -22,6 +22,10 @@ def initialize(options)
       options.has_key?(:cache_ok) ?
         options[:cache_ok] :
         true
+    @use_proxy =
+      options.has_key?(:use_proxy) ?
+        options[:use_proxy] :
+        true
```

Comment:
I would set this to default as `false`, as you mentioned in your PR, if you don't mind. That way any other users who aren't using a proxy won't have to make changes to their code. And since it's set on initialization, it should only need to be set once per instantiation anyway.